packages/es-components/src/components/patterns/datepicker/DatePicker.js

```diff
@@ -110,6 +158,14 @@ DatePicker.propTypes = {
   startDate: PropTypes.object,
   /** Sets the end date (moment) in a range */
   endDate: PropTypes.object,
+  /** The width of the window. Passed in from the withWindowSize higher order component */
+  windowWidth: PropTypes.number,
+  /**
+   * Determines whether to use the native datepicker instead of the React datepicker on mobile devices.
+   * For complicated scenarios like date ranges and such, it is recommended to disable this.
+   * Defaults to true.
+   */
+  allowNativeDatepickerOnMobile: PropTypes.bool,
```

Comment:
is it a real requirement to add this?

Reply:
It's one I gleaned from a conversation Ken told me about. Apparently we're worried the native date picker might not be ideal in every scenario. Like for date ranges and stuff like that.